src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -243,10 +243,11 @@ mkldnn::memory::format MKLDNNEmitter::query_convolution_forward_weight_format(
     mkldnn::memory::dims mkldnn_padding_below(padding_below.begin(), padding_below.end());
     mkldnn::memory::dims mkldnn_padding_above(padding_above.begin(), padding_above.end());
 
+    mkldnn::algorithm convolution_algo = mkldnn_utils::get_conv_algo();
```

Comment:
We call this repeatedly for each emitter function call. Would be nice to have a static variable instead and set it once upon emitter start.